MAUI/ImageEditor/migration.md

```diff
@@ -0,0 +1,487 @@
+</tr>
+
+<tr>
+<td>ImageEffect</td>
+<td>ImageEffect</td>
+<td>Specifies the image effect.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>ImageFileType</td>
+<td>Specifies the image file type.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>ImageCropType</td>
+<td>Specifies the image crop types.</td>
+</tr>
+</table>
+
+## Events
+
+<table>
+<tr>
+<th>Xamarin SfImageEditor</th>
+<th>.NET MAUI SfImageEditor</th>
+<th>Description</th></tr>
+
+<tr>
+<td>BeginReset</td>
+<td>BeginReset</td>
+<td>Occurs when the image reset action.</td>
+</tr>
+
+<tr>
+<td>ImageSaving</td>
+<td>ImageSaving</td>
+<td>Occurs when the image saving.</td>
+</tr>
+
+<tr>
+<td>ImageSaved</td>
+<td>ImageSaved</td>
+<td>Occurs when the image saved in local storage.</td>
+</tr>
+
+<tr>
+<td>ItemSelected</td>
+<td>AnnotationSelected</td>
+<td>Occurs when an annotation view selected.</td>
+</tr>
+
+<tr>
+<td>ImageLoaded</td>
+<td>ImageLoaded</td>
+<td>Occurs when image loaded into view initially.</td>
+</tr>
+</table>
+
+## Methods
+
+<table>
+<tr>
+<th>Xamarin SfImageEditor</th>
+<th>.NET MAUI SfImageEditor</th>
+<th>Description</th></tr>
+
+<tr>
+<td>Rotate</td>
+<td>Rotate</td>
+<td>Method to rotate image 90 degrees in clockwise direction.</td>
+</tr>
+
+<tr>
+<td>Flip</td>
+<td>Flip</td>
+<td>Method to reverses the image along its horizontal or vertical axis.</td>
+</tr>
+
+<tr>
+<td>Crop</td>
+<td>Crop</td>
+<td>Method to select and remove a portion of the image.</td>
+</tr>
+
+<tr>
+<td>AddText</td>
+<td>AddText</td>
+<td>Method to add text to the image.</td>
+</tr>
+
+<tr>
+<td>AddShape</td>
+<td>AddShape</td>
+<td>Method to add shapes such as rectangles, circles, and lines to the image.</td>
+</tr>
+
+<tr>
+<td>Save</td>
+<td>Save</td>
+<td>Method to save the edited image.</td>
+</tr>
+
+<tr>
+<td>SaveEdits</td>
+<td>SaveEdits</td>
+<td>Method to save the current editing actions.</td>
+</tr>
+
+<tr>
+<td>CancelEdits</td>
+<td>CancelEdits</td>
+<td>Method to cancel the editing actions.</td>
+</tr>
+
+<tr>
+<td>ClearAnnotations</td>
+<td>ClearAnnotations</td>
+<td>Method to remove all the annotations.</td>
+</tr>
+
+<tr>
+<td>Reset</td>
+<td>Reset</td>
+<td>Method to reset the edited image to its initial stage.</td>
+</tr>
+
+<tr>
+<td>Undo</td>
+<td>Undo</td>
+<td>Method to reverse or undo the last action or a series of actions.</td>
+</tr>
+
+<tr>
+<td>Redo</td>
+<td>Redo</td>
+<td>Method to replay or redo an action or a series of actions that were previously undone using the "Undo" command.</td>
+</tr>
+
+<tr>
+<td>Delete</td>
+<td>DeleteAnnotation</td>
+<td>Method to remove the current selected annotations.</td>
+</tr>
+
+<tr>
+<td>GetStream</td>
+<td>GetImageStream</td>
+<td>Method to get the edited image stream</td>
+</tr>
+
+<tr>
+<td>ApplyImageEffect</td>
+<td>ImageEffect</td>
+<td>Method to apply various visual effects to the image, such as adjusted brightness, blur, contrast, exposure, saturation, sharpen, hue, and opacity.</td>
+</tr>
+
+<tr>
+<td>Bounds</td>

```

Comment:
These should be added in properties under each class. Why these are added below methods

Reply:
changed